### ISSUE TITLE
Kodi repository

### ISSUE THREAD

**Issue by Cixert** [mrfixit2001/plugin.video.kodikaraoke#6]
Kodi Karaoke Free has disappeared from the G-Source repository. Is there another repository from which it can be installed and updated?
Is it possible to add this github as a repository source?
What would the URL be like?

**Comment by mrfixit2001** [comment]
Absolutely, I've added my own repository to keep this addon updated, you can get it from here:
https://mrfixit2001.github.io/
